packages/desktop-client/package.json

```diff
@@ -7,7 +7,7 @@
   ],
   "devDependencies": {
     "@juggle/resize-observer": "^3.4.0",
-    "@playwright/test": "^1.41.1",
+    "@playwright/test": "1.41.1",
```

Comment:
We are using this image in the pipeline:
```
image: mcr.microsoft.com/playwright:v1.41.1-jammy
```
Pinning here just keeps us in line with the pipeline version